browse/templates/base.html

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+
+<head>
+  {%- block head -%}
+  <title>{% block title %}{% endblock %}</title>
+  <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
+  <link rel="stylesheet" type="text/css" media="screen" href="//static.arxiv.org/css/arXiv.css?v=20170424" />
+  {# disable for now #}
+  {%- if 0 %}
+  <!-- Piwik -->
+  <script type="text/javascript">
+    var _paq = _paq || [];
+    _paq.push(["setDomains", ["*.arxiv.org"]]);
+    _paq.push(['trackPageView']);
+    _paq.push(['enableLinkTracking']);
+    (function() {
+      var u = "//webanalytics.library.cornell.edu/";
+      _paq.push(['setTrackerUrl', u + 'piwik.php']);
+      _paq.push(['setSiteId', 538]);
+      var d = document,
+        g = d.createElement('script'),
+        s = d.getElementsByTagName('script')[0];
+      g.type = 'text/javascript';
+      g.async = true;
+      g.defer = true;
+      g.src = u + 'piwik.js';
+      s.parentNode.insertBefore(g, s);
+    })();
+  </script>
+  <!-- End Piwik Code -->
+  {% endif -%}
+  {%- include "feedback_collector_js.html" -%}
+  {%- endblock head -%}
+</head>
+
+<body {% block body_id %}id="front"{% endblock %} class="with-cu-identity">
+  <noscript><img src="//webanalytics.library.cornell.edu/piwik.php?idsite=538&amp;rec=1" style="border:0;" alt="" /></noscript>
+
+  <div id="cu-identity">
+    <div id="cu-logo">
+      <a id="insignia-link" href="http://www.cornell.edu/"><img src="//static.arxiv.org/icons/cu/cul_signature_unstyled.gif" alt="Cornell University" width="283" height="76" border="0" /></a>
+      <div id="unit-signature-links">
+        <a id="cornell-link" href="http://www.cornell.edu/">Cornell University</a>
+        <a id="unit-link" href="http://www.library.cornell.edu/">Library</a>
+      </div>
+    </div>
+    <div id="support-ack">
+      <a href="{{url_for('acknowledgment')}}">We gratefully acknowledge support from<br/>the Simons Foundation<br /> and {{ session['institution'] if ('institution' in session and session['institution'] != None) else 'member institutions' }}</a>
+    </div>
+  </div>
+
+  <div id="header" {% block header_class %}{% endblock %}>
+    {% block header %}
+    <a href="{url_path('ignore_me')}"></a>
```

Comment:
Could we have a comment on this blank link saying what it's for? If it is visible to users/screen readers may need a class for it to explicitly be hidden or visible in some way.

Reply:
This is an old honeypot link that users ought to never see, but it will be good for us to check whether screen readers catch it. It's intended to trap robots/crawlers that are not respecting the contents of `robots.txt` (which indicates this link should not be crawled). I'll add a comment about it.